test/unit/TestIOSXRDriver.py

```diff
@@ -57,7 +59,9 @@ class FakeIOSXRDevice:
 
     @staticmethod
     def read_txt_file(filename):
-        with open(filename) as data_file:
+        curr_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
Because I didn't know such a thing!  I've always done it this way.  :-)  Learned something today, thanks @dbarrosop!